src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -19,8 +19,8 @@
   /**
    * Creates a new Intake.
    */
-  private WPI_TalonSRX intake_motor;
-  private WPI_TalonSRX wheel_motor;
+  public WPI_TalonSRX intake_motor;
+  public WPI_TalonSRX wheel_motor;
```

Comment:
No. Since these are hardware devices, we need to keep these private. Also, why? You don't access these anywhere...